src/swagger/oauth2-redirect.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
+<html lang="en-US">
+<head>
+    <title>Swagger UI: OAuth2 Redirect</title>
+</head>
+<body>
+</body>
+</html>
+<script>
+    'use strict';
+    function run () {
+        var oauth2 = window.opener.swaggerUIRedirectOauth2;
```

Comment:
I wonder if this html page is really necessary... Looks like a lot of javascript with some sort of authentification. To me that looks kind of scary on an otherwise static site like devdocs.

Reply:
I can remove it. I just grabbed everything in the `dist/` directory except the favicons.